apps/studio/components/interfaces/Connect/DatabaseConnectionString.tsx

```diff
@@ -0,0 +1,512 @@
+          {/* // handle non terminal examples */}
+          {hasCodeExamples && (
+            <div className="grid grid-cols-2 gap-20 w-full px-7 py-8">
+              <div>
+                <StepLabel number={++stepNumber} className="mb-4">
+                  Install the following
+                </StepLabel>
+                {exampleInstallCommands?.map((cmd, i) => (
+                  <CodeBlock
+                    key={i}
+                    className="[&_code]:text-[12px] [&_code]:text-foreground"
+                    value={cmd}
+                    hideLineNumbers
+                    language="bash"
+                  >
+                    {cmd}
+                  </CodeBlock>
+                ))}
+              </div>
+              {exampleFiles && exampleFiles?.length > 0 && (
+                <div className="">
+                  <StepLabel number={++stepNumber} className="mb-4">
+                    Add file to project
+                  </StepLabel>
+                  {exampleFiles?.map((file, i) => (
+                    <div key={i} className="">
+                      <CodeBlockFileHeader title={file.name} />
+                      <CodeBlock
+                        wrapperClassName="[&_pre]:max-h-40 [&_pre]:px-4 [&_pre]:py-3 [&_pre]:rounded-t-none"
+                        value={file.content}
+                        hideLineNumbers
+                        language={lang}
+                        className="[&_code]:text-[12px] [&_code]:text-foreground"
+                      />
+                    </div>
+                  ))}
+                </div>
+              )}
+            </div>
+          )}
+
+          <div>
+            {hasCodeExamples && (
+              <div className="px-7 pt-8">
+                <StepLabel number={++stepNumber}>Choose type of connection</StepLabel>
+              </div>
+            )}
+            <div className="divide-y divide-border-muted [&>div]:px-7 [&>div]:py-8">
+              <ConnectionPanel
+                contentType={contentType}
+                lang={lang}
+                type="direct"
+                title="Direct connection"
+                fileTitle={fileTitle}
+                description="Ideal for applications with persistent, long-lived connections, such as those running on virtual machines or long-standing containers."
+                connectionString={connectionStrings['direct'][selectedTab]}
+                onCopy={() => handleCopy(selectedTab)}
+                ipv4Status={{
+                  type: !ipv4Addon ? 'error' : 'success',
+                  title: !ipv4Addon ? 'Not IPv4 compatible' : 'IPv4 compatible',
+                  description: ipv4Addon && 'Connections are IPv4 proxied with IPv4 addon.',
+                  link: !ipv4Addon
+                    ? {
+                        text: 'IPv4 addon',
+                        url: `/project/${projectRef}/settings/addons?panel=ipv4`,
```

Comment:
This is using /project/ (without dashboard), which should be fine if we're using the Next.js <Link> component. However, the ConnectionPanel component uses a simple `a` tag, which leads to a missing `/dashboard` prefix in our deployed environment - we should probably just use the next.js link component in the ConnectionPanel component to render the links - then this can stay as is